### PR TITLE
Add livestream video to Prisma 4 upgrade guide

### PR DESCRIPTION
**Pull request by keerlu**

### CODE REVIEW

File: content/300-guides/300-upgrade-guides/200-upgrading-versions/030-upgrading-to-prisma-4.mdx [hunk 1]
```diff
@@ -470,3 +470,20 @@ yarn upgrade prisma@4 @prisma/client@4
 </tab>
 
 </TabbedContent>
+
+## Video guide
+
+For a video walkthrough of the upgrade process and examples of upgrade scenarios, see our livestream on upgrading to Prisma 4:
```
Comment (Jolg42) [inline content/300-guides/300-upgrade-guides/200-upgrading-versions/030-upgrading-to-prisma-4.mdx]:
```suggestion
For a video walkthrough of the upgrade process and examples of upgrade scenarios, see our recorded livestream on upgrading to Prisma 4:
```

File: content/300-guides/300-upgrade-guides/200-upgrading-versions/030-upgrading-to-prisma-4.mdx [hunk 2]
```diff
@@ -470,3 +470,20 @@ yarn upgrade prisma@4 @prisma/client@4
...
+
+<div class="videoWrapper">
+
+<iframe
+  width="560"
+  height="315"
+  src="https://www.youtube.com/embed/FSjkBrfaoEY"
+  frameborder="0"
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
```
Comment (Jolg42) [inline content/300-guides/300-upgrade-guides/200-upgrading-versions/030-upgrading-to-prisma-4.mdx:485]: accelerometer & gyroscope are super weird to me 🤔

**Review by Jolg42 — APPROVED** [review]

File: content/300-guides/300-upgrade-guides/200-upgrading-versions/030-upgrading-to-prisma-4.mdx (hunk 2 above)
Comment (keerlu) [reply content/300-guides/300-upgrade-guides/200-upgrading-versions/030-upgrading-to-prisma-4.mdx:485]: Hm, good point... this was copypasted from elsewhere in docs. Would you recommend removing it?

Comment (Jolg42) [reply content/300-guides/300-upgrade-guides/200-upgrading-versions/030-upgrading-to-prisma-4.mdx:485]: Looks like it's the default embed code and might be used by YouTube to rotate the video on mobile... Not sure exactly but let's keep it then.

Comment (keerlu) [reply content/300-guides/300-upgrade-guides/200-upgrading-versions/030-upgrading-to-prisma-4.mdx:485]: Actually it looks like that's part of Youtube's 'Embed' details when you hit 'Share'... maybe it's to enable detecting moving the phone from portrait to landscape? So seems fine.

**Pull request merged**